qa/src/test/java/io/camunda/migrator/qa/runtime/SkipAndRetryProcessInstancesTest.java

```diff
@@ -86,8 +86,7 @@ public void shouldSkipProcessWithMultiInstanceServiceTask() {
     // then
     assertThatProcessInstanceCountIsEqualTo(0);
 
-    List<IdKeyDbModel> skippedProcessInstanceIds =
-        idKeyMapper.findSkippedByType(IdKeyMapper.TYPE.RUNTIME_PROCESS_INSTANCE, 0 ,10).stream().toList();
+    List<IdKeyDbModel> skippedProcessInstanceIds = dbClient.findSkippedProcessInstances().stream().toList();
```

Comment:
The `idKeyMapper.findSkippedByType` is changed to `private` so `dbClient.findSkippedProcessInstances` is used here.